pkgs/http/lib/src/client.dart

```diff
@@ -25,6 +25,10 @@ import 'streamed_response.dart';
 /// [http.head], [http.get], [http.post], [http.put], [http.patch], or
 /// [http.delete] instead.
 ///
+/// All methods will emit a [ClientException] if there is a transport-level
+/// failure when communication with the server, e.g., if the server could not
```

Comment:
[nit] Avoid using "e.g."

https://dart.dev/effective-dart/documentation#avoid-abbreviations-and-acronyms-unless-they-are-obvious

Reply:
Done